### DISCUSSION TITLE
Make PointedDripstoneBlock LAVA and WATER cauldron fill probability changeable

### DISCUSSION THREAD

**Discussion by NitroxydeX** [PurpurMC/Purpur#864] [__Completed__] [answered]
This enhancement would be kinda like in the same spot as patch [0224](https://github.com/PurpurMC/Purpur/blob/ver/1.18.1/patches/server/0224-Cauldron-fill-chances.patch) with the cauldron fill chance.

In PointedDripstoneBlock there are two final floats WATER_CAULDRON_FILL_PROBABILITY_PER_RANDOM_TICK and LAVA_CAULDRON_FILL_PROBABILITY_PER_RANDOM_TICK - which are actually never used anywhere.

Mojang has hardcoded these values into the maybeFillCauldron function.

I know that this is still affected by the random tick itself but given the 5% chance for Lava and that the random tick appears relatively often per 20 ticks the fill chance will greatly improve when setting this to higher values than Mojangs defaults.

## Describe the solution you'd like.
Would be nice to have control over these fill probabilities in the config file like with rain and powder-snow

**Comment by npXXX** [comment]
+1
Would be great!

**Comment by encode42** [comment, marked as answer]
Was implemented in ad2bac1e243835f9eb09596bd1453e76e8845a0c! [Config options](https://purpurmc.org/docs/Configuration/#fill-chances)
